CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
 
 PROJECT(CiftiLib_and_example)
 
-SET(CMAKE_CXX_FLAGS "-W -Wall")
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -W -Wall")
```

Comment:
The fix I propose is actually pretty standard for compiler flags.

> Using the variable in its own definition looks a little disturbing, given that some invocations of SET are persistent.

See: https://cmake.org/Wiki/CMake_Performance_Tips, the `APPEND` section.